src/components/toolbox/Chart/chart.stories.js

```diff
@@ -0,0 +1,98 @@
+import Chart from './index';
+import { styles } from '../../../utils/chartOptions';
+import style from './chart.stories.css';
+
+storiesOf('Toolbox', module)
+  .add('Chart', () => (
+    <StoryWrapper>
+      <Box className={style.wrapper}>
+        <BoxHeader><h1>Line Chart</h1></BoxHeader>
+        <div className={style.container}>
+          <Chart
+            type="line"
+            data={{
+              labels: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Oug', 'Sep', 'Oct', 'Nov', 'Dec'],
+              datasets: [
+                {
+                  label: '123456789L',
+                  data: [35, 55, 23, 67, 98, 46, 134, 66, 70, 33, 100, 120],
+                  backgroundColor: styles.whiteSmoke,
+                  borderColor: styles.borderColor,
+                  pointBorderColor: styles.borderColor,
+                  pointBackgroundColor: styles.whiteColor,
+                  pointHoverBackgroundColor: styles.whiteColor,
+                  pointHoverBorderColor: styles.ultramarineBlue,
+                  pointHoverBorderWidth: 4,
+                  borderWidth: 2,
+                },
+              ],
+            }}
+          />
+        </div>
+      </Box>
+
+      <Box className={style.wrapper}>
+        <BoxHeader><h1>Bar Chart</h1></BoxHeader>
+        <div className={style.container}>
+          <Chart
+            type="bar"
+            data={{
+              labels: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Oug', 'Sep', 'Oct', 'Nov', 'Dec'],
+              datasets: [
+                {
+                  label: '123456789L',
+                  data: [35, 55, 23, 67, 98, 46, 134, 66, 70, 33, 100, 120],
+                  barPercentage: 1,
+                  categoryPercentage: 0.5,
+                  barThickness: 'flex',
+                  maxBarThickness: 14,
+                  minBarLength: 0,
+                },
+                {
+                  label: '987654321L',
+                  data: [10, 27, 30, 30, 80, 20, 100, 45, 138, 77, 88, 121],
+                  barPercentage: 1,
+                  categoryPercentage: 0.5,
+                  barThickness: 'flex',
+                  maxBarThickness: 14,
+                  minBarLength: 0,
+                  stack: 'dob',
+                  backgroundColor: '#2bd67b',
+                },
+              ],
+            }}
+          />
+        </div>
+      </Box>
+
+      <Box className={style.wrapper}>
+        <BoxHeader><h1>Doughnut Chart</h1></BoxHeader>
+        <div className={style.container}>
+          <Chart
+            type="doughnut"
+            data={{
+              labels: ['0 < 1', '1 < 100', '100 < 1K', '> 1K'],
+              datasets: [
+                {
+                  label: 'ABC',
+                  data: [100, 634, 329, 98],
+                  backgroundColor: [
+                    '#68d284',
+                    '#4873ec',
+                    '#2c4a93',
+                    '#0d152c',
+                  ],
```

Comment:
Similar comment to the above. Removing styles makes it look like this:
![Screenshot 2019-11-18 at 08 58 21](https://user-images.githubusercontent.com/1254342/69034808-5a45bc80-09e2-11ea-8baf-8c6fdbaf5fbe.png)

Reply:
@slaweet thx for the comments, about this comment and the other 2 above, I understand what you mentioned, however I was thinking in a way to do this but the only detail is that this visual especification is part of the **data** object not the **options** and for each chart we need to make a specific function to provide the data in the way we need to present the chart as design, that's why is looking like your mention

Reply:
You could do something like https://github.com/LiskHQ/lisk-desktop/pull/2638#discussion_r347306539

Reply:
Or, alternatively, you could put the colors into the `chartConstants.js` 
```js
export const doughnutPrimaryColorPallete = [
                    '#68d284',
                    '#4873ec',
                    '#2c4a93',
                    '#0d152c',
                  ],
```

and here just 
```js
import { doughnutPrimaryColorPallete } from '../../constants/chartConstants.js'
```

so that we don't have the definition of the colors in all files that use a chart.